db/migrate/20220725152514_status.rb

```diff
@@ -0,0 +1,5 @@
+class Status < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :doc_statuses, :doc_facility, index: true
```

Comment:
I think for all `add_reference` it's more important to have a primary_key than an index
Tip: search for `add_reference` in the existing code and check how it was implemented in other places (I do that for almost everything unless I'm 100% sure)

Reply:
```suggestion
    add_reference :doc_statuses, :doc_facility, foreign_key: true
```